client/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
I think we'd be better off dealing with the pain of enabling this. It'll force us to have complete and accurate type coverage at the cost of a higher learning curve for anyone newer to ts, but I think we can and should teach our way through that :)

Reply:
Agreed!